tests/models/test_flowcell_class.py

```diff
@@ -12,7 +12,7 @@ def test_get_run_parameters_when_non_existing(fixtures_dir: Path):
         / "apps"
         / "demultiplexing"
         / "demultiplexed-runs"
-        / "201203_A00689_0200_AHVKJCDRXX"
+        / "201203_D00483_0200_AHVKJCDRXX"
```

Comment:
Use comma instead of slash when building path

Reply:
fixed